function/spring-aggregator-function/src/test/java/org/springframework/cloud/fn/aggregator/CustomPropsAndMongoMessageStoreAggregatorTests.java

```diff
@@ -60,11 +60,8 @@ public void test() {
 
 		output.as(StepVerifier::create)
 			.assertNext((message) -> assertThat(message).extracting(Message::getPayload)
-				.isInstanceOf(List.class)
-				.asList()
-				.hasSize(1)
-				.element(0)
-				.isEqualTo("foo"))
+				.asInstanceOf(InstanceOfAssertFactories.LIST)
```

Comment:
As the PR comment describes, the `asList` is deprecated w/ suggested replacement `asInstanceOf`. I additionally simplified the usages as follows:
- `hasSize + element + isEqualTo` -> `containsExactly`
- `inInstanceOf + asList` -> `asInstanceOf(IOAF.LIST)` (it checks instancef implicitly)